### ISSUE TITLE
UX Request for Sorting

### ISSUE THREAD

**Issue by locuturus** [zhanghai/MaterialFiles#1319]
Please allow me to make as many changes in the sort menu as I want before applying them. If the menu stayed open until I tap away from it that would solve the problem as I see it. Currently switching the sort is frustrating because I probably want to change the sort type and the ascending or descending in the same moment but the menu closes after every change. It's a lot of extra taps in my opinion.

**Comment by zhanghai** [comment]
I agree, I do have plan to change it to a botom sheet so that its easier to toggle multiple 
things while it remains consistent for the expectation for the type of UX component being used. (Popup menus should be dismissed after interaction.)
